global/cookie-banner@3.0.js

```diff
@@ -71,5 +71,5 @@
     }
   )
 
-  setTimeout(loadCookieBanner, 3000);
+  setTimeout(loadCookieBanner, 1);
```

Comment:
@jethroflanagan is not worth just removing the setTimeout if it's like only 1ms?

Reply:
@michael616kriel It should stay as it needs to be removed from the initial render loop, just as a performance thing.